nautobot/docs/release-notes/version-2.4.md

```diff
@@ -166,6 +166,74 @@ As Python 3.8 has reached end-of-life, Nautobot 2.4 requires a minimum of Python
 
 <!-- towncrier release notes start -->
 
+## v2.4.8 (2025-05-12)
+
+### Security in v2.4.8
+
+- [#7223](https://github.com/nautobot/nautobot/issues/7223) - Updated dependency `h11` to `0.16.0` to address `CVE-2025-43859`. This is a development dependency and will not auto-update when upgrading Nautobot. Please be sure to update your local environment.
+- [#7273](https://github.com/nautobot/nautobot/issues/7273) - Updated `Django` to 4.2.21 to address `CVE-2025-32873`.
+
+### Added in v2.4.8
+
+- [#7048](https://github.com/nautobot/nautobot/issues/7048) - Added bulk-edit support for the Platform model.
+- [#7075](https://github.com/nautobot/nautobot/issues/7075) - Added Bulk Edit functionality for the webhook model.
+- [#7107](https://github.com/nautobot/nautobot/issues/7107) - Added bulk-edit support for the JobHook model.
+- [#7126](https://github.com/nautobot/nautobot/issues/7126) - Added Bulk Edit functionality for the CustomLink model.
+- [#7148](https://github.com/nautobot/nautobot/issues/7148) - Added Bulk Edit functionality for the RackGroup model.
```

Comment:
nit: can we standardize the structure and format of these? `bulk-edit` vs `Bulk Edit`, `support` vs ` functionality`, etc.?

Reply:
Opt for `Added Bulk Edit functionality for the <model_name> model.`